client/src/components/PacmanGame/gameCore.js

```diff
@@ -0,0 +1,78 @@
+export const initSquareGridState = edge => Array(edge).fill().map(() => Array(edge).fill(0));
+export const getFoods = () => [[15, 9], [14, 9], [13, 9], [12, 9], [11, 9], [10, 9]];
+
+const wallGenerator = (numberofCells) => {
+  const wall = [];
+  const int = num => parseInt(num, 10);
+
+  const generateVerticalWalls = () => {
```

Comment:
Can you return a wall array and use its result instead of mutating it? Or is there any particular reason?

Reply:
It's just a dummy function to generate an arbitrary wall pattern. Will be changed in the future.